src/main/java/org/broadinstitute/consent/http/service/DataAccessReportsParser.java

```diff
@@ -58,16 +63,18 @@ public void setDataSetApprovedUsersHeader(FileWriter darWriter) throws IOExcepti
                         HeaderDAR.RENEWAL_DATE.getValue() + END_OF_LINE);
     }
 
-    public void addApprovedDARLine(FileWriter darWriter, Election election, Document dar, String profileName, String institution, String consentName, String translatedUseRestriction) throws IOException {        String rusSummary = StringUtils.isNotEmpty( dar.getString(DarConstants.NON_TECH_RUS)) ?  dar.getString(DarConstants.NON_TECH_RUS).replace("\n", " ") : "";
+    public void addApprovedDARLine(FileWriter darWriter, Election election, DataAccessRequest dar, String profileName, String institution, String consentName, String translatedUseRestriction) throws IOException {       
+        String rusSummary = Objects.nonNull(dar.getData()) && StringUtils.isNotEmpty(dar.data.getNonTechRus()) ?  dar.data.getNonTechRus().replace("\n", " ") : "";
         String content1 =  profileName + DEFAULT_SEPARATOR + institution + DEFAULT_SEPARATOR;
+        String electionDate = (Objects.nonNull(election.getFinalVoteDate())) ? formatTimeToDate(election.getFinalVoteDate().getTime()) : "";
         String content2 = rusSummary + DEFAULT_SEPARATOR +
-                formatTimeToDate(new Date(dar.getLong(DarConstants.SORT_DATE)).getTime()) + DEFAULT_SEPARATOR +
-                formatTimeToDate(election.getFinalVoteDate().getTime()) + DEFAULT_SEPARATOR +
+                dar.getSortDate() + DEFAULT_SEPARATOR +
```

Comment:
use `formatTimeToDate` to convert the date to a String with a 'MM/DD/YYYY' format